File: tys-spa/ciudadano/src/app/domain/datosVinculacionPracticasAdolecente.ts
import {HorarioLaboral} from './horario-laboral';

export interface DatosVinculacionPracticasAdolecente {
  nombrePractica: string;
  practicaGratuita: boolean;
  objectoPractica: Array<any>;
  descripcionDetallada: string;
  horarioLaboral: HorarioLaboral;
  nombreEntidadInstitucion: string;
  formaEntrega: string;
  lugar: string;
  idPeriodicidadReconocimiento: number;
  idFormaEntrega: number;
}

File: tys-application/tys-commons/src/main/java/co/com/mintrabajo/tys/commons/domain/DatosVinculacionPracticasAdolecente.java
package co.com.mintrabajo.tys.commons.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

import java.util.List;

@Data
@AllArgsConstructor
@Builder(builderMethodName = "newInstance")
public class DatosVinculacionPracticasAdolecente {
    private String nombrePractica;
    private boolean practicaGratuita;
    private List<ObjetoPractica> objectoPractica;
    private String descripcionDetallada;
    private HorarioLaboral horarioLaboral;
    private String nombreEntidadInstitucion;
    private String formaEntrega;
    private Long idFormaEntrega;
    private String lugar;
    private Long idPeriodicidadReconocimiento;
}
